demos/default/pages/image.js

```diff
@@ -22,16 +22,6 @@ const Images = () => (
       />
       <Image src="https://i.imgur.com/bxSRS3Jb.png" alt="Tawny Frogmouth" width={160} height={160} />
     </p>
-
-    <p>
```

Comment:
A broken image in the images page is no longer required as this test is handled in `/broken-image` and associated tests for the default demo.

Reply:
Should the [import statement](https://github.com/netlify/next-runtime/pull/1722/files#diff-ddcb82dba113090c8615f81413148bfa58944bd2924992150a23ed6c83ed9a89R1) for `Image` in this file also be updated? Or is there a reason it's staying with `next/legacy/image` for the time being?

Reply:
Yes it should. Not sure how that got missed. 🤔